test/e2e/common_test.go

```diff
@@ -92,6 +116,68 @@ var _ = BeforeSuite(func() {
 		}
 		f.Close()
 	}
+	path, err := ioutil.TempDir("", "libpodlock")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	return []byte(path)
+}, func(data []byte) {
+	LockTmpDir = string(data)
+})
+
+func (p *PodmanTestIntegration) Setup() {
+	cwd, _ := os.Getwd()
+	INTEGRATION_ROOT = filepath.Join(cwd, "../../")
+	p.ArtifactPath = ARTIFACT_DIR
+}
+
+//var _ = BeforeSuite(func() {
```

Comment:
Not using this anymore?

Reply:
no but i want to keep the history for future changes with @cevich

Reply:
git does an awesome job of "keeping history", maybe just stick the former commit sha into a comment for reference?  Unless you have other nefarious purposes for this :smile:

Reply:
of course i do ... one step ahead!

Reply:
lol, okay, well they better be SUPER evil :smile: